src/app/components/spartan/ui-button-helm/src/lib/hlm-button.directive.ts

```diff
@@ -28,7 +29,7 @@ export const buttonVariants = cva(
 				sm: 'tw-py-[6px] tw-px-[20px] md:tw-rounded-[10px] tw-rounded-[7px] tw-text-[16px] tw-leading-[24px] tw-border tw-font-bold',
 				xs: 'tw-py-[4px] tw-px-[16px] md:tw-rounded-[10px] tw-rounded-[7px] tw-border',
 				xxs: 'tw-py-[3px] tw-px-[9px] md:tw-rounded-[10px] tw-rounded-[7px] tw-border',
-				icon: 'tw-h-10 tw-w-10',
+				icon: 'tw-h-12 tw-w-12 ',
```

Comment:
Did you check if this breaks other pages?

Reply:
yes. size="icon" is only used in `issuer-catalog.component.html` and nowhere else